docs/timeseries/data_access.rst

```diff
@@ -0,0 +1,194 @@
+.. _timeseries-data-access:
+
+Accessing data in time series
+*****************************
+
+.. |Time| replace:: :class:`~astropy.time.Time`
+.. |Table| replace:: :class:`~astropy.table.Table`
+.. |QTable| replace:: :class:`~astropy.table.QTable`
+.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
+.. |BinnedTimeSeries| replace:: :class:`~astropy.timeseries.BinnedTimeSeries`
+.. |time_attr| replace:: :attr:`~astropy.timeseries.TimeSeries.time`
+.. |time_bin_start| replace:: :attr:`~astropy.timeseries.BinnedTimeSeries.time_bin_start`
+.. |time_bin_center| replace:: :attr:`~astropy.timeseries.BinnedTimeSeries.time_bin_center`
+.. |time_bin_end| replace:: :attr:`~astropy.timeseries.BinnedTimeSeries.time_bin_end`
+.. |time_bin_size| replace:: :attr:`~astropy.timeseries.BinnedTimeSeries.time_bin_size`
+
+Accessing data
+==============
+
+For the examples in this page, we will consider a simple sampled time series
+with two data columns - ``flux`` and ``temp``::
+
+    >>> from collections import OrderedDict
+    >>> from astropy import units as u
+    >>> from astropy.timeseries import TimeSeries
+    >>> ts = TimeSeries(time='2016-03-22T12:30:31',
+    ...                        time_delta=3 * u.s,
+    ...                        data={'flux': [1., 4., 5., 3., 2.],
```

Comment:
I agree with @pllim here, as I was about to leave my own comment - definitely should have a unit for flux, to encourage such behavior since this is a `QTable`.  Simplar if `temp` is a temperature

Reply:
Fixed